### ISSUE TITLE
Improve error messages

### ISSUE THREAD

**Issue by DieMyst** [fluencelabs/aqua#75]
Here is an example:
```
4 func a() -> string:
5     r <- "a"
      ^=======
      Syntax error, expected: OneOfStr(65,List(<-, else, for, if, on, otherwise, par)), InRange(65,A,Z), InRange(70,a,z)
6     <- r
7 
```
The actual error is that `"a"` should be a function call or an ability with a function call.
What expected for example:
```
4 func a() -> string:
5     r <- "a"
              ^===
      Syntax error, expected: `function or service call`
6     <- r
7 
```

**Comment by coder11** [comment]
Should make a more clear message for Capital letter in method name

Aqua:
```
service Peer("peer"):
    is_connected: string -> bool

service Op("op"):
    identity: -> ()

data User:
    peer_id: string
    relay_id: string
    name: string

service Test("test"):
    getUserList: -> []User
    DoSomething: -> bool

func initAfterJoin(me: User) -> []User:
    allUsers <- Test.getUserList()
    for user <- allUsers par:
        on user.relay_id:
            isOnline <- Peer.is_connected(user.peer_id)
        if isOnline:
            on user.peer_id via user.relay_id:
                Test.doSomething()
        else:
            on user.peer_id via user.relay_id:
                Op.identity()
        par Test.doSomething()
    <- allUsers

```

Error:
```
12 service Test("test"):
13     getUserList: -> []User
14     DoSomething: -> bool
   ^========================
   Syntax error, expected: EndOfString(227,696)
15
16 func initAfterJoin(me: User) -> []User:
```

**Comment by coder11** [comment]
When `par` is in the wrong spot

Aqua:
```
service Peer("peer"):
    is_connected: string -> bool

service Op("op"):
    identity: -> ()

data User:
    peer_id: string
    relay_id: string
    name: string

service Test("test"):
    getUserList: -> []User
    doSomething: -> bool

func betterMessage(me: User) -> []User:
    allUsers <- Test.getUserList()
    for user <- allUsers par:
        on user.relay_id:
            isOnline <- Peer.is_connected(user.peer_id)
        if isOnline:
            on user.peer_id via user.relay_id:
                par Test.doSomething()
        Test.doSomething()
    <- allUsers
```

error:

```

18     for user <- allUsers par:
19         on user.relay_id:
20             isOnline <- Peer.is_connected(user.peer_id)
                                                          ^
                                                          Syntax error, expected: FailWith(445,Return type is defined for function, but nothing returned. Use `<- value` as the last expression inside function body.)
21         if isOnline:
22             on user.peer_id via user.relay_id:
```

**Comment by alari** [comment]
Duplicate of #40

**Comment by coder11** [comment]
https://github.com/fluencelabs/fluent-pad/compare/af59a0fe852c...ad362f57abab

This commits results in:

**Comment by coder11** [comment]
```
PS C:\work\fluence\minimal-aqua> npx aqua-cli --version

0.1.1-105
```

File content:

```
data EmptyServiceResult:
    ret_code: s32
    err_msg: string

```

Error message:

```
1 data EmptyServiceResult:
2     ret_code: s32
      ^=============
      Syntax error, expected: OneOfStr(40,List([], bool, f32, f64, i16, i32, i64, i8, string, u16, u32, u64, u8)), InRange(40,*,*), InRange(40,A,Z)
3     err_msg: string
4

```

**Comment by DieMyst** [comment]
> Should make a more clear message for Capital letter in method name
> 
> Aqua:
> 
> ```
> service Peer("peer"):
>     is_connected: string -> bool
> 
> service Op("op"):
>     identity: -> ()
> 
> data User:
>     peer_id: string
>     relay_id: string
>     name: string
> 
> service Test("test"):
>     getUserList: -> []User
>     DoSomething: -> bool
> 
> func initAfterJoin(me: User) -> []User:
>     allUsers <- Test.getUserList()
>     for user <- allUsers par:
>         on user.relay_id:
>             isOnline <- Peer.is_connected(user.peer_id)
>         if isOnline:
>             on user.peer_id via user.relay_id:
>                 Test.doSomething()
>         else:
>             on user.peer_id via user.relay_id:
>                 Op.identity()
>         par Test.doSomething()
>     <- allUsers
> ```
> 
> Error:
> 
> ```
> 12 service Test("test"):
> 13     getUserList: -> []User
> 14     DoSomething: -> bool
>    ^========================
>    Syntax error, expected: EndOfString(227,696)
> 15
> 16 func initAfterJoin(me: User) -> []User:
> ```

fixes in #111
